### PR TITLE
Add a skimr-v2 blog post

### PR DESCRIPTION
**Pull request by michaelquinn32**

### CODE REVIEW

**Comment by michaelquinn32** [comment]
I imagine we'll have to eventually update the dates to get everything coinciding with the push to cran. But please take a look at the current draft!

File: content/blog/2019-10-24-skimrv2.md [hunk 1]
```diff
@@ -0,0 +1,512 @@
+---
+slug: skimrv2
+title: "(Re)introducing `skimr` v2"
+subtitle: "A year in the life of an open source R project"
+package_version: 2.0
+authors:
+  - name: Michael Quinn
+    url: https://github.com/michaelquinn32
```
Comment (stefaniebutland) [inline content/blog/2019-10-24-skimrv2.md]:
```suggestion
```
new format for authors requires only this:

```
authors:
  - Michael Quinn
```

And your byline will link to your rOpenSci author page
https://ropensci.org/authors/michael-quinn/
https://ropensci.org/authors/elin-waring/

You can edit the links shown on your author page: [Instructions](https://github.com/ropensci/roweb2#create-an-authors-index-file-for-yourself-author-file)

File: content/blog/2019-10-24-skimrv2.md [hunk 2]
```diff
@@ -0,0 +1,512 @@
+---
+slug: skimrv2
+title: "(Re)introducing `skimr` v2"
+subtitle: "A year in the life of an open source R project"
+package_version: 2.0
+authors:
+  - name: Michael Quinn
```
Comment (stefaniebutland) [inline content/blog/2019-10-24-skimrv2.md]:
```suggestion
  - Michael Quinn
```

File: content/blog/2019-10-24-skimrv2.md [hunk 3]
```diff
@@ -0,0 +1,512 @@
...
+    url: https://github.com/michaelquinn32
+  - name: Elin Waring
```
Comment (stefaniebutland) [inline content/blog/2019-10-24-skimrv2.md]:
```suggestion
  - Elin Waring
```

File: content/blog/2019-10-24-skimrv2.md [hunk 4]
```diff
@@ -0,0 +1,512 @@
...
+    url: https://github.com/elinw
```
Comment (stefaniebutland) [inline content/blog/2019-10-24-skimrv2.md]:
```suggestion
```

File: content/blog/2019-10-24-skimrv2.md [hunk 5]
```diff
@@ -0,0 +1,512 @@
...
+  - software peer review
+  - R
+  - packages
+  - community
+  - skimr
+---
+
+[Theme song: *PSA* by Jay-Z](https://www.youtube.com/watch?v=-LzdKH1naok)
+
+We announced the testing version of `skimr` v2 on
```
Comment (stefaniebutland) [inline content/blog/2019-10-24-skimrv2.md]: no backticks for package name

`skimr` -> skimr throughout post

File: content/blog/2019-10-24-skimrv2.md [hunk 6]
```diff
@@ -0,0 +1,512 @@
+---
+slug: skimrv2
+title: "(Re)introducing `skimr` v2"
```
Comment (stefaniebutland) [inline content/blog/2019-10-24-skimrv2.md]:
```suggestion
title: "(Re)introducing skimr v2"
```

File: content/blog/2019-10-24-skimrv2.md [hunk 7]
```diff
@@ -0,0 +1,512 @@
...
+subtitle: "A year in the life of an open source R project"
+package_version: 2.0
+authors:
+  - name: Michael Quinn
+    url: https://github.com/michaelquinn32
+  - name: Elin Waring
+    url: https://github.com/elinw
+date: "2019-10-24"
```
Comment (stefaniebutland) [inline content/blog/2019-10-24-skimrv2.md]:
```suggestion
date: "2019-10-29"
```

File: content/blog/2019-10-24-skimrv2.md [hunk 8]
```diff
@@ -0,0 +1,512 @@
...
+time we pushed our first version to CRAN, Elin got `skimr`’s rOpenSci’s package
+[peer review](https://github.com/ropensci/software-review/issues/175) started
+(thank you Jennifer and Jim!), opening another incredibly useful channel for
+collecting feedback on the package. All of these new ideas and suggestions gave
+us the opportunity to really push `skimr` to the next level, but doing that
+would require rethinking the package, from the ground up.
+
+A month after finishing the peer review (and six months after the process
+began), we announced v2. Over the first phase of `skimr`’s life, we accumulated
+700 commits, two release, 400 GitHub stars, 95 percent code coverage and a
```
Comment (stefaniebutland) [inline content/blog/2019-10-24-skimrv2.md]:
```suggestion
700 commits, two releases, 400 GitHub stars, 95 percent code coverage and a
```
-> releases

File: content/blog/2019-10-24-skimrv2.md [hunk 9]
```diff
@@ -0,0 +1,512 @@
...
+  - skimr
+---
+
+[Theme song: *PSA* by Jay-Z](https://www.youtube.com/watch?v=-LzdKH1naok)
+
+We announced the testing version of `skimr` v2 on
+[June 19, 2018](https://github.com/ropensci/skimr/issues/341). After more than a
+year of (admittedly intermittent) work, we’re thrilled to be able to say that
+the package is ready to go to CRAN. So, what happened over the last year? And
+why are we so excited for V2?
```
Comment (stefaniebutland) [inline content/blog/2019-10-24-skimrv2.md]:
```suggestion
why are we so excited for v2?
```

**Comment by stefaniebutland** [comment]
@michaelquinn32 @elinw Can you make those edits for publication & tweets Tues Oct 29?

**Comment by michaelquinn32** [comment]
Thanks for the excellent review! I really appreciate it.

For specific comments above:

* I think we can go tomorrow (10/29)
* I added a bit more information about learning more about the package at the bottom. Please let me know if there's a better location for that
* I made some manual changes to the HTML of the table, since the "header" class seemed to blow up the size of the text. Let me know what you think!

**Comment by stefaniebutland** [comment]
@michaelquinn32 Just realized the post doesn't say what skimr is! Can you please add that as close to the start as possible?

With that, I can publish the post. Thank you for the edits!

**Comment by michaelquinn32** [comment]
Great point! Moved the introduction there. Please let me know if there's anything else to catch. I'm quite happy to see how much it's improved after a couple of drafts.

**Comment by stefaniebutland** [comment]
Perfect! I'll publish and tweet shortly.
You are a generous blog post author @michaelquinn32 ;-) The post rocks!

**Comment by michaelquinn32** [comment]
Thanks Stefanie!

**Pull request merged**